src/AppBundle/Service/ReportStatusService.php

```diff
@@ -5,14 +5,20 @@
 use AppBundle\Entity\Report\Report;
 use JMS\Serializer\Annotation as JMS;
 
+/**
+ * Statuses are cached into report.statusCached, and used when present
+ * The cached status are set from the endpoints on CRUD operations on sections
+ * Look at `ReportStatusUpdaterCommand` and its cron usage.
+ *
+ */
 class ReportStatusService
 {
     const STATE_NOT_STARTED = 'not-started';
     const STATE_INCOMPLETE = 'incomplete';
     const STATE_DONE = 'done';
     const STATE_NOT_MATCHING = 'not-matching'; //only used for balance section
     const STATE_EXPLAINED = 'explained'; //only used for balance section
-
+    const ENABLE_SECTION_STATUS_DB_CACHE = true;
```

Comment:
if this is a feature flag it should be named the same as the feature: ie ENABLE_STATUS_CACHE